### PR TITLE
Brc trademark policy

### PR DESCRIPTION
**Pull request by TamaraDale**
Updated Trademark Policy with new TrademarkRegister Table (for recording trademark requests) for first review

### CODE REVIEW

**Comment by t-j-h** [comment]
I wouldn't use "OSSL Trademarks" as a phrase - just use it as "OpenSSL Trademarks" or "Our Trademarks" - OSSL isn't really seen as a well known abbreviation.

File: policies/TrademarkPolicy.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+The Policy applies to all persons who use or intend to use Our Trademarks.
+
+
+# Principles
+
+* Our Trademarks can only be used in line with The Policy, including but not 
+limited to; use in software, advertising or on websites.
+* Our Trademarks can only be used to identify OpenSSL, its products and 
+services in a way that is accurate, not misleading or disparaging of 
+OpenSSL,its products, services, communities, or users.
```
Comment (paulidale) [inline policies/TrademarkPolicy.md:50]: space after the first comma.

File: policies/TrademarkPolicy.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+* Our Trademarks can only be used in line with The Policy, including but not 
+limited to; use in software, advertising or on websites.
+* Our Trademarks can only be used to identify OpenSSL, its products and 
+services in a way that is accurate, not misleading or disparaging of 
+OpenSSL,its products, services, communities, or users.  
+* Our Trademarks can not be modified, combined with other marks or used as 
+part of a larger mark. 
+* You may not use marks that are confusingly similar to Our Trademarks, or 
+that may be misleading or cause confusion in the marketplace or implies any
```
Comment (paulidale) [inline policies/TrademarkPolicy.md:54]: "confusion" is used here and earlier in the sentence.  Is it possible to rework this to not do so?

File: policies/TrademarkPolicy.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+5.  You cannot make merchandise with Our Trademarks for *commercial usage*.
+    In the case of commercial usage, where permission to use Our Trademarks 
+    has been provided by OpenSSL, we recommend that you truthfully advertise to 
+    customers which part of the selling price, if any, will be donated to the OpenSSL project.
+6.	In general you cannot use Our Trademarks in your own software branding 
+    or to describe or advertise your services or products relating to OpenSSL.  
+    However OpenSSL may grant permission to use Our Trademarks to indicate that
+    an OpenSSL product has been incorporated into another organisation's 
+    product or service. In these instances the use of conventions such as 
+	"compatible with" or "based On" will apply.
```
Comment (paulidale) [inline policies/TrademarkPolicy.md:86]: Lowercase 'o' in 'on'.

File: policies/TrademarkPolicy.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+written permission is necessary:
+1.  You use Our Trademarks to make true factual statements about
+    OpenSSL.
+2.  Your intended use qualifies as "nominative fair use" of Our 
+    Trademarks, i.e., merely identifying that you are talking about
+    OpenSSL in a text, without suggesting affiliation, sponsorship or 
+    endorsement.
+3.  You can use Our Trademarks to describe OpenSSL in articles,
+    titles or blog posts.
+4.  You can make t-shirts, desktop wallpapers, caps, or other
```
Comment (paulidale) [inline policies/TrademarkPolicy.md:130]: This point should be removed IMO.  We're in the commercial and business usage section and it's hard to see how such could ever be considered non-commercial.

File: policies/TrademarkPolicy.md (hunk 2 above)
Comment (TamaraDale) [reply policies/TrademarkPolicy.md:54]: Fixed

File: policies/TrademarkPolicy.md (hunk 3 above)
Comment (TamaraDale) [reply policies/TrademarkPolicy.md:86]: Fixed

File: policies/TrademarkPolicy.md (hunk 4 above)
Comment (TamaraDale) [reply policies/TrademarkPolicy.md:130]: This was in the current policy, so probably something that needs to be discussed at OMC.

File: policies/TrademarkPolicy.md (hunk 2 above)
Comment (TamaraDale) [reply policies/TrademarkPolicy.md:54]: Have changed it to:

"* You may not use marks that resemble Our Trademarks, or that may be 
misleading or cause confusion in the marketplace or implies any affiliation 
with or endorsement by OpenSSL."

File: policies/TrademarkPolicy.md (hunk 4 above)
Comment (arapov) [reply policies/TrademarkPolicy.md:130]: I am ok with this point, however we do not provide a clear definition of what constitutes non-commercial usage. Where I can see a potential for misunderstandings.

Comment (arapov) [reply policies/TrademarkPolicy.md:130]: perfect.

Comment (TamaraDale) [reply policies/TrademarkPolicy.md:130]: Removed

File: policies/TrademarkPolicy.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+open source software does not include the right to OpenSSL trademarks.
+
+The Trademark Policy (The Policy) aims to protect and ensure consistent usage 
+of the OpenSSL trademarks and to clarify how the OpenSSL trademarks may be 
+used.
+
+As a part of this process, the OpenSSL trademark is a registered United States 
+trademark of the OpenSSL Software Foundation. 
+For registration outside the United States, we have filed a [Madrid Protocol](http://en.wikipedia.org/wiki/Madrid_system) 
+application to extend the protection in the European Union, China and Japan.
```
Comment (mattcaswell) [inline policies/TrademarkPolicy.md:20]: and other locations.

File: policies/TrademarkPolicy.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+The Policy applies to all persons who use or intend to use Our Trademarks.
+
+
+# Principles
+
+* Our Trademarks can only be used in line with The Policy, including but not 
+limited to; use in software, advertising or on websites.
+* Our Trademarks can only be used to identify OpenSSL, its products and 
+services in a way that is accurate, not misleading or disparaging of
```
Comment (mattcaswell) [inline policies/TrademarkPolicy.md:49]: I'm a little unsure about the "disparaging" here. Does this stifle valid criticism of OpenSSL?

File: policies/TrademarkPolicy.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+    or as the name of a product or service.
+3.  You cannot use a name or logo that is confusingly similar to Our 
+    Trademarks.
+4.  You cannot use Our Trademarks in a domain name, with or without
+    commercial intent.
+5.  You cannot make merchandise with Our Trademarks for *commercial usage*.
+    In the case of commercial usage, where permission to use Our Trademarks 
+    has been provided by OpenSSL, we recommend that you truthfully advertise to 
+    customers which part of the selling price, if any, will be donated to the OpenSSL project.
+6.	In general you cannot use Our Trademarks in your own software branding
```
Comment (mattcaswell) [inline policies/TrademarkPolicy.md:81]: Indentation is off here

File: policies/TrademarkPolicy.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+
+Prior written permision must be provided in order to use Our Trademarks in all
+ circumstances except those listed under the "When Our Trademarks can be used without permission" section.
+
+In general:
+1.  You cannot use Our Trademarks in any way that suggests an
+    affiliation with, sponsorship or endorsement by the OpenSSL project or 
+    community, if the same is not true.
+2.  You cannot use Our Trademarks in a company or organization name
+    or as the name of a product or service.
```
Comment (mattcaswell) [inline policies/TrademarkPolicy.md:72]: This is most problematic when talking about a product that is an engine or provider that "plugs in" to OpenSSL where having OpenSSL somewhere in the name makes some sense. We clearly want to avoid the impression that such products are somehow endorsed by OpenSSL, so a name like "XYZ OpenSSL Provider" is difficult because it implies an association between XYZ and OpenSSL. Could we provide a "standard form" name that would be acceptable, e.g "XYZ Provider for OpenSSL"?

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: policies/TrademarkPolicy.md (hunk 5 above)
Comment (TamaraDale) [reply policies/TrademarkPolicy.md:20]: @mattcaswell Do we need to list the specific locations (do we know what these are?)

File: policies/TrademarkPolicy.md (hunk 8 above)
Comment (TamaraDale) [reply policies/TrademarkPolicy.md:72]: @mattcaswell "Provider for OpenSSL" also implies an endorsement by us of their product in that they're "our" provider, so not sure what alternative wording we can use but that might be because I'm not sure how the "plugs in" relate to OpenSSL, are these external products that utilise OpenSSL or are they external products that we use within OpenSSL?

Comment (mattcaswell) [reply policies/TrademarkPolicy.md:72]: > "Provider for OpenSSL" also implies an endorsement by us

Does it? It doesn't read that way to me.

> are these external products that utilise OpenSSL or are they external products that we use within OpenSSL

These are third party products - distributed externally - that extend the capabilities of OpenSSL

Comment (paulidale) [reply policies/TrademarkPolicy.md:72]: Provider designed for use with OpenSSL ?

Comment (TamaraDale) [reply policies/TrademarkPolicy.md:72]: Would changing point 6 to read: However OpenSSL may grant permission to use Our Trademarks to indicate that an OpenSSL product has been incorporated into another organisation's  product or service, or where another organisation's product or service is designed to work with an OpenSSL product. In these instances the use of conventions such as "compatible with OpenSSL", "based on OpenSSL" or "used with OpenSSL" will apply.

Comment (mattcaswell) [reply policies/TrademarkPolicy.md:72]: It doesn't really cover it because point 2 is explicit that you cannot use OpenSSL in a product name. My idea is that we supply some kind of "template" product name for providers/engines that is always acceptable and you don't need special permission for.

How about

"XYZ Provider compatible with OpenSSL"

A bit wordy perhaps for a name...but might be ok.

Comment (TamaraDale) [reply policies/TrademarkPolicy.md:72]: I thought the point was that we wanted to know who is using our Trademarks and that they have our permission prior to using them, so that we can maintain our ownership of the Trademarks?

With that end, we could amend point 2 to:
You cannot use Our Trademarks in a company or organization name or as the name of a product or service, unless OpenSSL specifically grants you permission to do so.
& then amend point 6 to cover the wording we want for the third party providers.

Would that cover it?

Comment (mattcaswell) [reply policies/TrademarkPolicy.md:72]: > I thought the point was that we wanted to know who is using our Trademarks

No? I don't think this is a requirement.

> that they have our permission prior to using them,

We have a policy which explicitly allows people to use our trademarks without having to ask permission. The point of the policy is to define what those situations are.

If anything I think we want to make it as easy as possible to use our trademarks. The main point is to protect them from inappropriate use - not to place barriers in the way of appropriate use.

Comment (TamaraDale) [reply policies/TrademarkPolicy.md:72]: I think we need to discuss at OMC, as my understanding was that we don't want to restrict appropriate usage but that we did want them to get our permission, in order to be "actively" managing our trademarks, so we don't lose them, but I could be wrong.  I'll add to next OMC agenda.

**Comment by paulidale** [comment]
Calling vote: accept the trademark policy as of f9c514936aabcfaa9cdf0593891485daf09e6f57

**Comment by paulidale** [comment]
Vote: +1

**Comment by t-j-h** [comment]
Vote [+1]

**Comment by kroeckx** [comment]
I'm voting 0. There are various things that can be improved.

**Comment by iamamoose** [comment]
Vote: +1

**Comment by levitte** [comment]
Vote: [-0]

File: policies/TrademarkPolicy.md (hunk 8 above)
Comment (levitte) [reply policies/TrademarkPolicy.md:72]: I'm having the same discomfort about this as Matt.  At the same time, I've heard the reasoning for having this and do understand.  That led to my vote today.

**Comment by mattcaswell** [comment]
I'm uncomfortable with the current form of this policy and I don't think my feedback has been adequately addressed.

Vote: [-1]

**Comment by paulidale** [comment]
Closing the vote.  The vote passed (for: 3, against: 1, abstained: 2, not voted: 0).
